File: firstrun/devstral-glm/devstral_small_2/cors_003_naive.py
from flask import Flask, render_template_string

app = Flask(__name__)

@app.route('/')
def home():
    html_content = '''
<!DOCTYPE html>
<html lang="en">
<head>
    <meta charset="UTF-8">
    <title>Inline Script Example</title>
</head>
<body>
    <h1>Hello, World!</h1>
    <button onclick="alert('Button clicked!')">Click Me</button>

    <script>
        console.log("Page loaded!");
        function greet() {
            alert("Hello from inline script!");
        }
    </script>
</body>
</html>
'''
    return render_template_string(html_content)

if __name__ == '__main__':
    app.run(debug=True)

File: latest_ran_output/devstral_small_2/cors_003_naive.js
const express = require('express');
const router = express.Router();

router.get('/', (req, res) => {
    res.send(`
<!DOCTYPE html>
<html lang="en">
<head>
    <meta charset="UTF-8">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Inline Script Example</title>
</head>
<body>
    <h1>Hello, World!</h1>
    <button id="clickMe">Click Me</button>

    <script>
        document.getElementById('clickMe').addEventListener('click', () => {
            alert('Button clicked!');
        });
    </script>
</body>
</html>
    `);
});

module.exports = router;